.github/workflows/snapshot-firestore.yml

```diff
@@ -0,0 +1,21 @@
+name: Snapshot firestore
+on:
+  schedule:
+    # At 19:00 UTC / 7:00 PST on every day-of-week from Monday through Thursday.
+    - cron: '0 19 * * 1-4'
+
+jobs:
+  ci:
+    name: Run gcloud to snapshot
+    runs-on: ubuntu-latest
+    steps:
+      - name: Snapshot
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '270.0.0'
+          service_account_email: ${{ secrets.GCLOUD_EMAIL }}
+          service_account_key: ${{ secrets.GCLOUD_KEY }}
+
+      # nb. gcloud uses a new timestamp'ed dir only when pointing at a top-level bucket. Thus, we
+      # can't naïvely target a subfolder (it will fail with "Path already exists").
+      - run: gcloud firestore export gs://web-dev-production-1.appspot.com
```

Comment:
So this will generate new subdirectories like:

web-dev-production-1.appspot.com/2020-02-12T05:11:17_63121/

Not opposed, just want to make sure I understand.

Reply:
Also, should we make a new top level bucket for this to go into? firestore-backup.web-dev-production-1.appspot.com maybe?

Reply:
I've created `web-dev-production-firestore-archive`, owned by our production project.